website/docs/project_settings/settings_project_global.md

```diff
@@ -13,6 +13,25 @@ Project settings can have project specific values. Each new project is using stu
 Projects always use default project values unless they have [project override](../admin_settings#project-overrides) (orage colour). Any changes in default project may affect all existing projects.
 :::
 
+## Color Management (ImageIO)
+
+:::info Default OCIO config
+OpenPype is distributing own ocio configs. Those can be found in `{openpype install dir}/{version}/vendor/bin/ocioconfig/OpenColorIOConfigs`. Windows example: `C:\Program Files (x86)\OpenPype\3.14.0\vendor\bin\ocioconfig\OpenColorIOConfigs`
+:::
+
+### Using OCIO config
+Each path input is supporting [anatomy template key formating](../admin_settings_project_anatomy#available-template-keys) and also environment variables. Default distributed value is set to `{OPENPYPE_ROOT}/vendor/bin/ocioconfig/OpenColorIOConfigs/aces_1.2/config.ocio`. If any of host config is enabled and set to any path, global config path is overriden.
+
+Config path can be defined to particular shot target with following path input `{root[work]}/{project[name]}/{hierarchy}/{asset}/config/aces.ocio`
```

Comment:
Does it need to be for each shot? Or can you define it on a sequence level?

Reply:
> Should this maybe additionally mention that when enabled like this EACH shot must have a config/aces.ocio of its own? (Does that make sense in production?)

@BigRoy Yeah, now when you mention it - I believe it will not be really solid feature for some. I presume some sort of falback, or multi path inputs with order should be implemented. I guess best approach on this would be to add all templated paths targeting all hierarchical paretns. So for example. 

1. `{root[work]}/{project[name]}/{hierarchy}/{asset}/config/aces.ocio`
2. `{root[work]}/{project[name]}/{hierarchy}/config/aces.ocio`
3. `{root[work]}/{project[name]}/config/aces.ocio`

Only that 2nd path will be a bit tricky. All parents will need to be iterated for possible paths to be tested.

Reply:
> Does it need to be for each shot? Or can you define it on a sequence level?

@tokejepsen I guess the approach mentioned above would be addressing what you thing, right?

Reply:
> @tokejepsen I guess the approach mentioned above would be addressing what you thing, right?

Yeah, I think going up the hierarchy from the current asset/version/representation would make sense and make it flexible enough for production.